docs/interface/json-rpc.md

```diff
@@ -70,15 +70,15 @@ Response:
 {"jsonrpc":"2.0","result":true,"id":1}
 ```
 
-#### getBalance
+#### balance
```

Comment:
I think the documentation in `docs/` is deprecated

Reply:
That's correct. We should simply remove that folder.